src/main/resources/application.conf

```diff
@@ -33,6 +33,11 @@ docker {
   // dockerToken = ""
 }
 
+jobavoidance {
+  // Set to true in order to avoid running previously run jobs.
+  enabled = false
+}
+
```

Comment:
I like defaults too but I though here that forcing the user to specify this conf was good in order to make sure that they're aware of what's going to happen regarding job avoidance.
If that's not a concern we can make this optional but then we should mention in the config file that it's optional (like for the docker conf for example)